src/codeu/chat/client/commandline/Chat.java

```diff
@@ -308,7 +318,7 @@ private ConversationContext find(String title) {
     //
     panel.register("info", new Panel.Command() {
       @Override
-      public void invoke(Scanner args) {
+      public void invoke(List<String> args) {
```

Comment:
This is not compiling for me. Line 128 has an extra `panel.register("info", new Panel.Command() {` that is using the Scanner. This most likely came in when you merged in master to resolve the conflicts. I'm pasting the error below (:

```
src/codeu/chat/client/commandline/Chat.java:128: error: <anonymous codeu.chat.client.commandline.Chat$2> is not abstract and does not override abstract method invoke(List<String>) in Command
    panel.register("info", new Panel.Command() {
                                               ^
src/codeu/chat/client/commandline/Chat.java:129: error: method does not override or implement a method from a supertype
      @Override
      ^
```